matlab/schemas/+vis/SingleDot.m

```diff
@@ -0,0 +1,18 @@
+%{
+vis.SingleDot (manual) # single dot to map receptive field
+-> vis.Condition
+-----
+luminance                   : float                         # cd/m^2
+contrast                    : float                         # Michelson's 0-1
```

Comment:
what is this the contrast of?  If you specify the brightness and contrast, then what do the `dot_color` and `bg_color` indicate?  Why is color 0-255?